### PR TITLE
add support for kicad exporting

### PR DESCRIPTION
**Pull request by rcooke-warwick**
Change-type: patch
Signed-off-by: Ryan Cooke <ryan@balena.io>

### CODE REVIEW

**Comment by rcooke-warwick** [comment]
@Bucknalla

**Comment by Bucknalla** [comment]
@ntzovanis

**Comment by Bucknalla** [comment]
@rcooke-warwick we can update to v2.2 of KiCAD-Export which allows for a one-liner expressing all of the config files

**Comment by rcooke-warwick** [comment]
> @rcooke-warwick we can update to v2.2 of KiCAD-Export which allows for a one-liner expressing all of the config files

done @Bucknalla

File: hardware/kicad-exports.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ARGV_DIRECTORY="$1"
+set -u
+
+[[ "${DEBUG}" == "false" ]] || set -x
+
+cd $ARGV_DIRECTORY
+
+# export bill of materials
+echo -e "exporting bill of materials"
+/export.sh -c /opt/kibot/config/*.yaml -e /source/*.sch -d ../outputs/
```
Comment (Bucknalla) [inline hardware/kicad-exports.sh:13]: @rcooke-warwick does this work? I haven't tried it but very neat if it does

**Review by nazrhom — APPROVED** [review]

**Review by nazrhom — APPROVED** [review]

**Pull request merged**